Lib/test/test_typing.py

```diff
@@ -3915,6 +3927,53 @@ def blah():
 
         blah()
 
+    def set_up_overloads(self):
+        def blah():
+            pass
+
+        overload1 = blah
+        overload(blah)
+
+        def blah():
+            pass
+
+        overload2 = blah
+        overload(blah)
+
+        def blah():
+            pass
+
+        return blah, [overload1, overload2]
+
+    # Make sure we don't clear the global overload registry
+    @patch("typing._overload_registry",
+        defaultdict(lambda: defaultdict(dict)))
+    def test_overload_registry(self):
```

Comment:
Can we add a test to make sure that an entry isn't added into the registry if `get_overloads` is called on a function that doesn't have any overloads?

Reply:
debbf8acd09000f91ef04b64b21475cda094f68a

Reply:
debbf8acd09000f91ef04b64b21475cda094f68a to you too